{'activation': 'relu', 'dropout': 0.6273329672, 'hist_agg': 'none', 'layers': 4, 'learning_method': 'rmsprop', 'lr': 0.0078662021, 'n_cases': 16643, 'n_features': 305, 'n_games': 6, 'steps': 800, 'units': 87}
Cannot create random sample of size 24964 because after dropping Nan from y only 23639 cases remained.
['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 21, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 791, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 770, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


{'activation': 'tanh', 'dropout': 0.3279981047, 'hist_agg': 'none', 'layers': 5, 'learning_method': 'sgd', 'lr': 0.0055670142, 'n_cases': 26500, 'n_features': 205, 'n_games': 4, 'steps': 200, 'units': 80}
Cannot create random sample of size 39750 because after dropping Nan from y only 24438 cases remained.
['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 21, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 791, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 770, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


{'activation': 'tanh', 'dropout': 0.3887124665, 'hist_agg': 'none', 'layers': 3, 'learning_method': 'sgd', 'lr': 0.0084934939, 'n_cases': 25568, 'n_features': 105, 'n_games': 2, 'steps': 300, 'units': 92}
Cannot create random sample of size 38352 because after dropping Nan from y only 25261 cases remained.
['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 21, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 791, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 770, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


{'activation': 'tanh', 'dropout': 0.3327688812, 'hist_agg': 'none', 'layers': 2, 'learning_method': 'sgd', 'lr': 0.0074221889, 'n_cases': 17630, 'n_features': 255, 'n_games': 5, 'steps': 600, 'units': 91}
Cannot create random sample of size 26445 because after dropping Nan from y only 24034 cases remained.
['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 21, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 791, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 770, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


{'activation': 'tanh', 'dropout': 0.4345172056, 'hist_agg': 'none', 'layers': 3, 'learning_method': 'sgd', 'lr': 0.006258435, 'n_cases': 10201, 'n_features': 255, 'n_games': 5, 'steps': 400, 'units': 82}

['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 21, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 791, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 770, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


{'activation': 'tanh', 'dropout': 0.5581166361, 'hist_agg': 'none', 'layers': 2, 'learning_method': 'sgd', 'lr': 0.0080600732, 'n_cases': 10185, 'n_features': 255, 'n_games': 5, 'steps': 800, 'units': 51}

['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 21, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 791, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 770, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


{'activation': 'tanh', 'dropout': 0.3499657812, 'hist_agg': 'none', 'layers': 2, 'learning_method': 'sgd', 'lr': 0.0077518652, 'n_cases': 10592, 'n_features': 255, 'n_games': 5, 'steps': 400, 'units': 64}

['  File "scripts/meval.sc", line 3, in <module>\n    do_meval.process_cmd_line()\n', '  File "/home/delano/working/python_misc/ledona/__init__.py", line 21, in wrapper\n    result = timed_func(*args, **kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 564, in process_cmd_line\n    result = calc_engine.run_evaluation(eval_func)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 791, in run_evaluation\n    calc_def_id=calc_def_id)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/engine.py", line 770, in calculate\n    **extra_kwargs)\n', '  File "/home/delano/working/fantasy/fantasy_py/calculation/regression/exe/do_meval.py", line 472, in eval_func\n    \'db\': meta_info.db_path.split(os.sep)[-1]})\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 955, in search\n    n_calls=n_calls, n_random_starts=n_random_starts)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/gp.py", line 228, in gp_minimize\n    callback=callback, n_jobs=n_jobs)\n', '  File "/home/delano/working/fantasy/venv-3.6/lib/python3.6/site-packages/skopt/optimizer/base.py", line 248, in base_minimize\n    next_y = func(next_x)\n', '  File "/home/delano/working/fantasy/fantasy_py/util/hyper_param_tuning.py", line 917, in skopt_func\n    self.errors[flattened_hp_dict] = (ex, traceback.format_stack())\n']
----------------------------------------


